### DISCUSSION TITLE
Mac/OSX alternative to Vircom

### DISCUSSION THREAD

**Discussion by nicwats** [wills106/homeassistant-solax-modbus#112] [General]
As per the title, anyone have any idea for a Mac based alternative to Vircom? I used to be able to dual boot but have a load of issues trying to install windows to an external drive which I haven't gotten around yet so hoping an OSX based version exists but can't find it.

**Comment by wills106** [comment]
Do you need to use  Vircom to do the initial setup to get the Waveshare onto the network before you can use the Web GUI?

**Reply by nicwats** [reply]
Yeah, I may just have to dig out an old windows laptop or something
